torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -27,6 +27,7 @@
 )
 from torch._prims_common import CUDARngStateHelper
 from torch._subclasses import FakeTensor
+from torch.distributed._functional_collectives import AsyncCollectiveTensor
```

Comment:
oh @IvanKobzarev global import of `torch.distributed` won't work because it assumes that distributed has been built.

If we don't want to worry about extra runtime overhead of making the import local / happen at runtime, you could do:
    if torch.distributed.is_available():
        async_collective_type = type(torch.distributed._functional_collectives.AsyncCollectiveTensor)
    else:
        # dummy type, we know we won't have any AsyncCollectiveTensors at runtime if distributed is not available
        async_collective_type = NoneType